src/storage/src/redis_sets.cc

```diff
@@ -682,6 +682,14 @@ rocksdb::Status Redis::SMembers(const Slice& key, std::vector<std::string>* memb
       SetsMemberKey sets_member_key(key, version, Slice());
       Slice prefix = sets_member_key.EncodeSeekKey();
       KeyStatisticsDurationGuard guard(this, DataType::kSets, key.ToString());
+
+      // read_options.fill_cache = false;
```

Comment:
smember这里的read_options不需要设置fill_cache，上下边界值吗？

Reply:
there some other functions with `default_option` rather than `read_option` with bound and fill_cache.  I **only** wrote `read_option` in a little bit functions.

还有一些函数使用 `default_option` 而不是有上下界的 `read_option`，我只给部分函数增加了 `read_option`。